o1vm/src/mips/tests.rs

```diff
@@ -148,8 +148,11 @@ mod unit {
             // Only 8kb of memory (two PAGE_ADDRESS_SIZE)
             memory: vec![
                 // Read/write memory
-                (0, vec![0; PAGE_SIZE as usize]),
-                // Executable memory. Allocating 4 * 4kB
+                // Initializing with random data
+                (
+                    0,
+                    (0..PAGE_SIZE).map(|_| rng.gen_range(0u8..=255)).collect(),
+                ), // Executable memory. Allocating 4 * 4kB
```

Comment:
Your plugin tries to align after the `,`. I think it should not be the case.

Reply:
Oh, fun because I didnt run the 80-char align command on that file this time... will fix it manually